lib/Dialect/SV/Transforms/PrettifyVerilog.cpp

```diff
@@ -111,8 +111,9 @@ bool PrettifyVerilogPass::prettifyUnaryOperator(Operation *op) {
   //         = ^tmp2 + 42
   //
   // This is particularly helpful when the operand of the unary op has multiple
-  // uses as well.
-  if (op->use_empty() || op->hasOneUse())
```

Comment:
I might rather try preserving the name when duplicating and letting the code which cares decide if it needs to spill.  This function is really important to making uses in different blocks look good.  (e.g. a use at module scope, inside always blocks, inside if blocks).